### PR TITLE
Additional psi4 attributes for v2

### PR DESCRIPTION
**Pull request by amandadumi**
This removes some print statements and expands psi4 coverage to the missing attributes. Cleans up some print unneeded statements. 

Marking this as WIP so i can work on other attributes and rebase against #1424 once merged.

### CODE REVIEW

File: cclib/attribute_parsers/mosyms.py
Comment (shivupa) [inline cclib/attribute_parsers/mosyms.py:189]: printed search and replace issue?

**Review by shivupa — APPROVED** [review]
LGTM after minor comment fix

File: cclib/attribute_parsers/scftargets.py
Comment (berquist) [inline cclib/attribute_parsers/scftargets.py:79]:
```suggestion
                this_scftargets = ccdata.scftargets
```
Does this bug exist in the original code?

**Review by berquist — CHANGES_REQUESTED** [review]

**Comment by berquist** [comment]
Also good for rebase

File: cclib/attribute_parsers/scftargets.py
Comment (amandadumi) [reply cclib/attribute_parsers/scftargets.py:79]: haha no i doubt it, probably just trying to go too quickly on my end.

Comment (amandadumi) [reply cclib/attribute_parsers/scftargets.py]: This is fixed now!

**Review by berquist — APPROVED** [review]
Sorry I missed that this was ready before I did the merge.  I'll rebase and merge.

**Comment by amandadumi** [comment]
Oh, thanks for doing that!!

**Pull request merged**
